### DISCUSSION TITLE
Custom columns?

### DISCUSSION THREAD

**Discussion by f1ow03** [audacious-media-player/audacious#1382] [Q&A] [answered]
Couldn't find a feature to create a custom column. Is there such feature in Audacious or should I write a plugin for it?

**Comment by Flaconia** [comment]
@f1ow03  Did you find a way to do it? I suggest opening an issue in the repo in case you haven't.
In my case I'd like to add a custom column to show the file formats and another one to activate/deactivate the song being played, like in AIMP.

**Comment by radioactiveman** [comment, marked as answer]
The Qt interface allows to configure them by right-clicking on the playlist header.

![Qt](https://github.com/user-attachments/assets/c442684e-824f-424c-99b7-45312516cebd)


The GTK interface allows to configure the visible playlist column at "Settings -> Appearance -> Playlist Columns".

![GTK](https://github.com/user-attachments/assets/3e0e071b-807c-4077-aea9-4033d2dac5e0)

Other colums are not supported yet.

**Reply by f1ow03** [reply]
I see. I can only use what's in the list. Thanks!

**Comment by cunlem** [comment]
Hmm so showing Composer is not possible not good
